cfep-07.md

```diff
@@ -0,0 +1,112 @@
+
+<table>
+<tr><td> Title </td><td> Migration strategy to Anaconda compilers </td>
+<tr><td> Status </td><td> Proposed </td></tr>
+<tr><td> Author(s) </td><td> Marius van Niekerk &lt;@mariusvniekerk&gt;</td></tr>
+<tr><td> Created </td><td> Aug 8, 2018</td></tr>
+<tr><td> Updated </td><td> Aug 8, 2018</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda-forge/conda-forge-enhancement-proposals/pull/10 </td></tr>
+<tr><td> Implementation </td><td> (listed below) </td></tr>
+</table>
+
+## Abstract
+
+As part of the move to the Anaconda compilers we need to replace some parts of our tooling whilst not
+affecting the rest of the conda-forge build stack.
+
+To do this we will publish to two labels (main and gcc7)
+
+## Implementation
+
+Implmentation of this is done in phases to avoid breaking the majority of the conda forge stack.
+
+As part of this we are introducing a few system keys into conda-forge-pinning
+
+*   channel_sources
+
+    This is a list of channels that are allowed to be used as sources for building conda-forge packages
+    
+    ```yaml
+    channel_sources:
+    - conda-forge,defaults              # [compiler_label < 7]
+    - conda-forge/label/gcc7,defaults   # [compiler_label >= 7]
```

Comment:
`conda-forge` not being here means that we have to build pure python packages, meta packages etc.